packages/replayio/src/utils/installation/installLatestRelease.ts

```diff
@@ -11,82 +11,91 @@ import { debug } from "./debug";
 import { getLatestRelease } from "./getLatestReleases";
 import { MetadataJSON } from "./types";
 
+function getPendingDownloadMessage(attempt: number) {
```

Comment:
This method should probably also accept the `maxAttempts` param. Else it would be easy to change the 5 below and leave these two values in conflict.